src/main/resources/init_db.sql

```diff
@@ -18,3 +18,29 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
+CREATE TABLE taxi_service_db.cars (
+		`id` BIGINT NOT NULL AUTO_INCREMENT,
+        `model` VARCHAR(255) NULL,
+        `manufacturer_id` BIGINT NULL,
+        `is_deleted` TINYINT NOT NULL DEFAULT 0,
+        PRIMARY KEY (`id`),
+        CONSTRAINT `cars_manufacturer_fk`
+									FOREIGN KEY (`manufacturer_id`)
+                                    REFERENCES `taxi_service_db`.`manufacturers` (`id`)
```

Comment:
```suggestion
                                    REFERENCES manufacturers` (`id`)
```
And here
And the others places